### PR TITLE
Add board support for Winterbloom Big Honking Button

### PR DESCRIPTION
**Pull request by theacodes**

### CODE REVIEW

File: ports/atmel-samd/boards/winterbloom_big_honking_button/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+CIRCUITPY_GAMEPAD = 0
+CIRCUITPY_I2CSLAVE = 0
+CIRCUITPY_NETWORK = 0
+CIRCUITPY_TOUCHIO = 0
+CIRCUITPY_PS2IO = 0
+CIRCUITPY_USB_HID = 0
+CIRCUITPY_RTC = 0
+
+CFLAGS_INLINE_LIMIT = 60
+SUPEROPT_GC = 0
```
Comment (tannewt) [inline ports/atmel-samd/boards/winterbloom_big_honking_button/mpconfigboard.mk]: If you have the space, then you can remove these two lines and maybe get a bit of speed back.

File: ports/atmel-samd/boards/winterbloom_big_honking_button/pins.c [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#include "shared-bindings/board/__init__.h"
+
+STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
+    { MP_ROM_QSTR(MP_QSTR_BUTTON), MP_ROM_PTR(&pin_PA07) },
+    { MP_ROM_QSTR(MP_QSTR_HONK_OUT), MP_ROM_PTR(&pin_PA02) },
```
Comment (tannewt) [inline ports/atmel-samd/boards/winterbloom_big_honking_button/pins.c]: Lol! Love it.

**Comment by tannewt** [comment]
Looks like the test wants you to alphabetize the board in build.yml.

File: ports/atmel-samd/boards/winterbloom_big_honking_button/mpconfigboard.mk (hunk 1 above)
Comment (theacodes) [reply ports/atmel-samd/boards/winterbloom_big_honking_button/mpconfigboard.mk]: Cool, will do.

**Comment by theacodes** [comment]
Thanks for the tips and review, @tannewt!

**Pull request merged**